docker/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+
+ # SPDX-License-Identifier: Apache-2.0
+ # Licensed to the Ed-Fi Alliance under one or more agreements.
+ # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+ # See the LICENSE and NOTICES files in the project root for more information.
+
+
+version: "3"
+
+services:
+  amt-service:
+    container_name: amt-container
+    restart: always
+    build: .
+    links:
+      - mssql-service
+      - pg-service
+
+  mssql-service:
+    container_name: amt-mssql
+    image: mcr.microsoft.com/mssql/server:2017-GA-ubuntu
+    restart: always
+    ports:
+      - 1433:1433
+    env_file:
+      - ./mssql-vars.env
+
+  pg-service:
+    container_name: amt-pg
+    image: postgres:11-alpine
+    ports:
+      - 5432:5432
+    env_file:
+      - ./postgres-vars.env
```

Comment:
I think it would be simpler if you have just one `.env` file instead of two. And then you won't need to use the `env_file` setting in this YML file, as the `.env` file will be read automatically.

Reply:
However, you would need add an `environment` section for each of MS and PG. For example:

```yml
    environment:
      POSTGRES_USER: "${POSTGRES_USER}"
      POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}"
      POSTGRES_DB: "postgres"
      PG_PORT: 5432
```

I've never specified the `DATABASE_HOST` property before in other docker setups for PostgreSQL.